Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Controllers/BrakeController.cs

```diff
@@ -313,8 +313,20 @@ public void Parse(string lowercasetoken, STFReader stf)
                             float value = stf.ReadFloat(STFReader.UNITS.None, null);
                             int smooth = stf.ReadInt(null);
                             string type = stf.ReadString();
-                            Notches.Add(new MSTSNotch(value, smooth, type, stf));
-                            if (type != ")") stf.SkipRestOfBlock();
+                            string name = null;
+                            while(type != ")" && !stf.EndOfBlock())
+                            {
+                                switch (stf.ReadItem().ToLower())
+                                {
+                                    case "(":
+                                        stf.SkipRestOfBlock();
+                                        break;
+                                    case "ortslabel":
+                                        name = stf.ReadStringBlock(null);
```

Comment:
This is not a bug, but worth noting, stf.ReadStringBlock will read whitespace delimited, so only the first word of a label will be picked up unless the label is enclosed in quotation marks. (Eg: ORTSLabel ( Initial Reduction ) will produce name = "Initial", while ORTSLabel ( "Initial Reduction" ) will produce name = "Initial Reduction".)

The solution here is to just encourage users to use quotation marks in their labels, else they will only capture the first word. Also worth noting that [this post](http://www.elvastower.com/forums/index.php?/topic/37445-dynamic-brake-features/page__view__findpost__p__300693) would not function as intended. People have a habit of finding old posts in Elvas Tower and assuming any code shown is actually how things work, so it may be best to edit this post to use quotation marks so it's a better example.

Reply:
Thanks, I inserted quotation marks